Formula/kcli.rb

```diff
@@ -0,0 +1,111 @@
+class Kcli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Management tool for libvirt/aws/gcp/kubevirt/openstack/ovirt/vsphere/packet"
+  homepage "https://kcli.readthedocs.io/en/latest/"
+  url "https://github.com/karmab/kcli/archive/326734afa7cd0704a8a95854e7f30686ff51b4f9.tar.gz"
+  sha256 "ba48b5dff2226499b1f8a28f318f95471cb19bf140240f80be9bee74ead8f49b"
```

Comment:
Can you also kindly ask upstream to do a tagged a release?
```suggestion
  url "https://github.com/karmab/kcli/archive/326734afa7cd0704a8a95854e7f30686ff51b4f9.tar.gz"
  version "99.0.202206301135"
  sha256 "ba48b5dff2226499b1f8a28f318f95471cb19bf140240f80be9bee74ead8f49b"
```

Reply:
@karmab this is for you :D

Reply:
> Can you also kindly ask upstream to do a tagged a release?

They created a tag :) Thanks!